sdk/modelsrepository/Azure.IoT.ModelsRepository/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.0.0-preview.3 (Unreleased)
+## 1.0.0-preview.3 (2021-04-12)
```

Comment:
preview.4?

Reply:
No, this is the 3rd one, This file is automatically updated when we release a version.

Reply:
And in this case it has been bumped correctly.